packages/gatsby-theme-docs/src/templates/page-content.js

```diff
@@ -66,7 +78,7 @@ export const query = graphql`
       isGlobalBeta
       excludeFromSearchIndex
       body
-      tableOfContents(maxDepth: 4)
+      tableOfContents(maxDepth: 6)
```

Comment:
```suggestion
      tableOfContents
```

Since we're using all header levels